prerequisites/install-functions/report_results.sh

```diff
@@ -15,7 +15,7 @@ report_results()
 
   type_CMAKE=`type ${CMAKE}`
   fully_qualified_CMAKE="/${type_CMAKE#*/}"
-  cmake_install_path="${fully_qualified_CMAKE%%/cmake*}"
+  cmake_install_path="${fully_qualified_CMAKE%%bin/cmake*}"
```

Comment:
Reviewing the commit that introduced this bug, 924d80a634bb3094ceee9273e122277d3fd88a96, there is no obvious reason for the [greedy substring removal](http://wiki.bash-hackers.org/syntax/pe#from_the_end) and I think that simply removing one of the two `%` may be a cleaner approach.